modules/sap_hana/main.tf

```diff
@@ -96,7 +96,7 @@ resource "google_compute_instance" "gcp_sap_hana" {
     sap_hana_sidadm_uid        = "${var.sap_hana_sidadm_uid}"
     sap_hana_sapsys_gid        = "${var.sap_hana_sapsys_gid}"
 
-    startup-script = "${var.startup_script}"
+    startup-script = "${file(var.startup_script)}"
```

Comment:
Instead of breaking the module interface, could we move this into the example code? So the module itself still accepts the actual startup script (not a file path)?